doc/files/package.json.md

```diff
@@ -108,8 +107,34 @@ You can check [the full list of SPDX license IDs](https://spdx.org/licenses/).
 Ideally you should pick one that is
 [OSI](http://opensource.org/licenses/alphabetical) approved.
 
-It's also a good idea to include a LICENSE file at the top level in
-your package.
+If your package is licensed under multiple common licenses, use an SPDX license
+expression syntax version 2.0 string, like this:
+
+    { "license" : "(ISC OR GPL-3.0)" }
+
+If you are using a license that hasn't been assigned an SPDX identifier, or if
+you are using an uncommon or custom license, do not include a "license" string
+in package.json. In those cases especially, but also more generally, it's a good
+idea to include a LICENSE file at the top level of the package.
```

Comment:
I think it may make sense to do some special filtering of the warnings reported by `init-package-json` in npm itself to do something commonsense, like looking for "see LICENSE file" (or some other, bikesheddable phrase) in that field and then verifying that `./LICENSE` does in fact exist. I've been meaning to do some work on how those warnings are handled for a while now, because I'm not sure it's useful to print the warnings for dependencies on install.

I don't think those changes need to be part of landing this change, though. We can get to them later. It just means that npm's install warnings are going to be noisier than usual for a while.